setup.py

```diff
@@ -28,6 +28,10 @@
     ],
     packages=[NAME],
     include_package_data=False,
-    install_requires=["Click", "Jinja2"],
+    install_requires=[
+        "Click==7.1.2", 
+        "Jinja2==2.11.2",
+        "pyperclip==1.8.0",
+    ],
```

Comment:
This is probably what I was missing to get pyperclip running in tox.  I only added the -c option to work around this limitation (I was rushing to write my article at the time. :) )   Do you have an opinion on removing the -c and just making it always copy to the clipboard?

Reply:
Funny you say that, I kind of wondered why it needed an option. Was going to write a bash wrapper to put it on all the time for myself anyhow :)